From a9005f512f3c9db3ecbf7b0b15c816126f2420b1 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 3 May 2013 14:57:37 +0000
Subject: [PATCH 110/462] Mantis 0023546: ServiceApp: Flash GUI use velocity
 for SVC   $JarVelocity   $JarVelocityTools   $JarApacheCollections schema
 generated source in ${GenSrcDir}

${PlatformJavaDir}/CLAApplet replaces ${AppJavaDir2}/LaunchScreen

git-svn-id: svn://localhost/SelfServiceBuild/trunk@163 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 3 +++
 SelfServiceBuild/Build/java/AntTargets.xml    | 5 +++--
 SelfServiceBuild/Build/java/build.xml         | 4 ++--
 3 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 670c218..4167e82 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -169,6 +169,9 @@
 <property name="JarXerces"                  value="${DIR_GLOBALS}/ThirdParty/xerces-j/1.4.0/xerces-j.jar"/>
 <property name="JarXMLBeans"                value="${DIR_GLOBALS}/ThirdParty/XMLBeans/xmlbeans-2.5.0/lib/xbean.jar"/>
 <property name="JarJSR173"                  value="${DIR_GLOBALS}/ThirdParty/XMLBeans/xmlbeans-2.5.0/lib/jsr173_1.0_api.jar"/>
+<property name="JarVelocity"                value="${DIR_GLOBALS}/ThirdParty/velocity/1.7/velocity-1.7.jar"/>
+<property name="JarVelocityTools"           value="${DIR_GLOBALS}/ThirdParty/velocity-tools/2.0/lib/velocity-tools-generic-2.0.jar"/>
+<property name="JarApacheCollections"       value="${DIR_GLOBALS}/ThirdParty/velocity/1.7/lib/commons-collections-3.2.1.jar"/>
 
 
 <!-- JRE -->
diff --git a/SelfServiceBuild/Build/java/AntTargets.xml b/SelfServiceBuild/Build/java/AntTargets.xml
index c124333..81313fb 100644
--- a/SelfServiceBuild/Build/java/AntTargets.xml
+++ b/SelfServiceBuild/Build/java/AntTargets.xml
@@ -86,15 +86,16 @@
     <basename property="GenFile" file="${SchemaFile}" suffix=".xsd"/>    
     <!-- Die angegebene Schema-Datei wird compiliert. -->
     <echo message="Compiling XML-Beans Schema: ${SchemaFile}"/>
-    <xmlbean schema="${SchemaFile}" destfile="${JarDir}/${GenFile}.jar" classpath="${JarXMLBeans};${JarJSR173}"/>
+    <xmlbean schema="${SchemaFile}" srcgendir="${GenSrcDir}" destfile="${JarDir}/${GenFile}.jar" classpath="${JarXMLBeans};${JarJSR173}"/>
 </target>
 
 <!-- create-xml-bean ================================================= -->
 
 <target name="create-xml-bean" if="project.xmlBean">
+    <mkdir dir="${GenSrcDir}"/>
+
     <!-- Alle Schema-Dateien werden compiliert. -->
     <foreach list="${project.xmlBean}" delimiter=" " param="SchemaFile" target="create-xml-bean-file" inheritall="true"/>
-
 </target>
 
 
diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index 7637bbe..07915ed 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -24,9 +24,9 @@
 
     <!-- Definition der Verzeichnisse mit den Java-Modulen ======= -->
     <property name="CommonDirList"   value="${CommonJavaDir}/OpenORB-1.3.1/OpenORB"/>
-    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/Tools,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings"/>
+    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/Tools,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/CLAApplet"/>
     <property name="AblsDirList"     value="${AblsJavaDir}/KioskProxy,${AblsJavaDir}/TunnelServer"/>
-    <property name="AppDirList"      value="${AppJavaDir2}/LaunchScreen"/>
+    <property name="AppDirList"      value=""/>
 
     <!-- compile-javagroup ======================================= -->
 
-- 
2.41.0.windows.1

